content/source/docs/cloud/sentinel/manage-policies.html.md

```diff
@@ -64,18 +64,66 @@ Terraform Cloud has support for Sentinel's modules feature. This allows you to w
+
+```sentinel
+import "http"
+import "json"
+import "strings"
+
+uri = "https://raw.githubusercontent.com/dmfilipenko/timezones.json/master/timezones.json"
+request = http.get(uri)
+response = json.unmarshal(request.body)
+
+offset = func(abbr) {
+	timezone = filter response as _, r {
+		r.abbr is strings.to_upper(abbr)
+	}
+	for timezone as tz {
+		print("Getting timezone data for", tz.value)
+		print("Details:", tz.text)
+		print("Abbreviation:", tz.abbr)
+		print("Offset:", tz.offset)
+		return tz.offset
+	}
+}
+```
 
--> **NOTE:** At this point in time, you cannot load modules from outside of policy set working directory hierarchy. This means in the above example, a `source` of `../modules/foo.sentinel` will not work.
+The above configuration would tell a policy check to load the code at `./modules/timezone.sentinel` relative to the policy set working directory and make it available to be imported with the statement `import "timezone"` at the top of your Sentinel policy code. This module will be available to all of the policies within the policy set.
+
+-> **NOTE:** At this point in time, you cannot load modules from outside of policy set working directory hierarchy. This means in the above example, a `source` of `../modules/timezone.sentinel` will not work.
 
 ### Sentinel policy code files
 
-Sentinel policies themselves are defined in individual files (one per policy) in the same directory as the `sentinel.hcl` file. These files must match the name of the policy from the configuration file and carry the `.sentinel` suffix. Using the configuration example above, a policy file named `sunny-day.sentinel` should also exist alongside the `sentinel.hcl` file to complete the policy set.
+Sentinel policies themselves are defined in individual files (one per policy) in the same directory as the `sentinel.hcl` file. These files must match the name of the policy from the configuration file and carry the `.sentinel` suffix. Using the configuration example above, a policy file named `enforce-terraform-maintenance-windows.sentinel` should also exist alongside the `sentinel.hcl` file to complete the policy set. 
+
+An example configuration for the `enforce-terraform-maintenance-windows.sentinel` policy would be similar to the following:
+
+```sentinel
+import "time"
+import "tfrun"
+import "timezone"
+
+param maintenance_days default ["Friday", "Saturday", "Sunday"]
+param timezone_abbreviation default "PST"
+
+tfrun_created_at = time.load(tfrun.created_at)
+
+supported_maintenance_day = rule {
+	tfrun_created_at.add(time.hour * timezone.offset(timezone_abbreviation)).weekday_name in maintenance_days
+}
+
+main = rule {
+	supported_maintenance_day
+}
+```
+
+-> **NOTE:** The above examples uses parameters to to facilitate module reuse within Terraform. For more information on parameters, see the [Sentinel parameter documentation](https://docs.hashicorp.com/sentinel/language/parameters/).
```

Comment:
```suggestion
-> **NOTE:** The above examples uses parameters to facilitate module reuse within Terraform Cloud. For more information on parameters, see the [Sentinel parameter documentation](https://docs.hashicorp.com/sentinel/language/parameters/).
```

It seems as though parameter documentation is limited to the API for Terraform Cloud. It would be good to note for the future that a Terraform Cloud specific guide to parameters would be useful.